source/_integrations/sleepiq.markdown

```diff
@@ -17,6 +22,8 @@ ha_platforms:
   - binary_sensor
   - button
   - light
+  - number
```

Comment:
The number entity is not added in the linked PR, thus this change seems unrelated?

Reply:
I noticed a couple missing categories/platforms so I figured I'd add them all at once (number was added earlier this month). I can do that in a separate PR if needed

Reply:
I can do it here instead: https://github.com/home-assistant/home-assistant.io/pull/21925